react/week4/github_user_search/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
I know you didn't learn about testing, but just as a challenge, can you make this pass?

Reply:
Sorry kasper, I was still thinking. Please answer me how to make this pass.

Reply:
may be everything should be before render??

Reply:
Thank you for your time and reviewing briefly and comments,I will make changes according to your comments kasper,. 
 :)

Reply:
@soujanyapolavarapu You don't have to make this pass at all. It was just if you wanted to. The React course didn't teach about testing. Sorry if it was confusing :)